src/Vehicle/Vehicle.h

```diff
@@ -752,6 +752,8 @@ class Vehicle : public FactGroup
     void _setHomePosition(QGeoCoordinate& homeCoord);
     void _setMaxProtoVersion (unsigned version);
 
+    QList<LinkInterface*>       links() { return _links; }
```

Comment:
Really needed? I assume private. If public I don't really want vehicle links to bleed out because there is some reference counting stuff that could get screwed up if they hand out to other parts of code.